src/utils/i18n.ts

```
@@ -4,6 +4,8 @@ import i18next from 'i18next';
 import path from 'path';
 import { fileURLToPath } from 'url';
 
+const __dirname = path.resolve(process.cwd(), 'src', 'utils');
+
 const languages = ['en', 'pt-BR', 'es'];
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Biome (2.1.2)

[error] Shouldn't redeclare '__dirname'. Consider to delete it or rename it.

'__dirname' is defined here:


(lint/suspicious/noRedeclare)